.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: python
 sudo: false
-
+matrix:
+  include:
+    - python: 3.5
+      env: TOXENV=py35
 env:
   - TOXENV=py27
   - TOXENV=py33
   - TOXENV=py34
   - TOXENV=pypy
   - TOXENV=pypy3
```

Comment:
FWIW I think https://pypi.python.org/pypi/tox-travis could let you simplify this to just
```yaml
language: python
sudo: false
python:
  - 2.7
  - 3.3
  - 3.4
  - 3.5
  - pypy
  - pypy3
install:
  - pip install tox-travis
script:
  - tox
notifications:
    email: false
```

As a bonus, the Travis build pages would show the Python version for each job, instead of the current "No language set".